### ISSUE TITLE
Ping teams for marker teams

### ISSUE THREAD

**Issue by XAMPPRocky** [rust-lang/triagebot#345]
Right now we have rustbot ping everyone individually (`cc @ghost`) in a marker team, this isn't scable as GitHub mentions stop after 50 people. This might seem hard to reach but [ICE-Breakers Cleanup Crew][icc] has 24 members and [ICE-Breakers LLVM][llvm] has 14 members. possible that we get more than 50 members, or if there are issues we want to tag multiple marker teams, we would easily hit the over 50 limit.

We could add logic to triagebot to only a select 50 people at a time, but then we'd need to have logic to ensure everyone gets an equal amount of notifications, and would add significant complexity. It would be simpler to have them as github teams. This also has the added benefit of the notification going into the `team-mentions` section in the new GitHub notifications rather than `mention`. Since these issues don't require that individuals specific attention rustbot should respect that and ping the team.

@pietroalbini mentioned in #3 reservations about granting marker team members with a GitHub Team since they would be granted "member" giving some kind of perceived authority. I don't think we should be applying any kind of authority to GitHub's structure. We already have members on GitHub that don't match with our current membership. `rust-lang/team` should be considered the normative structure of the Rust teams.

https://github.com/rust-lang/rust/issues/69225#issuecomment-586941472
[icc]: https://github.com/rust-lang/team/blob/master/teams/icebreakers-cleanup-crew.toml
[llvm]: https://github.com/rust-lang/team/blob/master/teams/icebreakers-llvm.toml

**Comment by pietroalbini** [comment]
> mentioned in #3 reservations about granting marker team members with a GitHub Team since they would be granted "member" giving some kind of perceived authority. I don't think we should be applying any kind of authority to GitHub's structure. We already have members on GitHub that don't match with our current membership. rust-lang/team should be considered the normative structure of the Rust teams.

To clarify, my worry here is not about *us* considering people inside the org members of the team. I'm worried about someone who's a member of the org but not in a team posting a controversial comment on the issue tracker, and that comment being picked up by social media. In that case the comment appears to be from a generic member of the organization, and it could be hard to convince people that person was not affiliated with the project.

**Comment by Mark-Simulacrum** [comment]
Yes, I think it may be a far-fetched concern but until we get closer to this being a problem in practice I would prefer not to worry about it. I think it may also be true that we can avoid the "noise" issue by editing the comment after posting to delete the pings, and potentially (I forget if GH appropriately does this) we could bypass the 50 people limit by editing the comment to mention more people -- I think I've seen this work at least sometimes.
